.github/workflows/test-plugins-against-cockpit.yml

```
@@ -16,22 +17,89 @@ jobs:
     timeout-minutes: 30
     runs-on: ubuntu-22.04
     outputs:
-      shell_versions: ${{ steps.collect-shell-versions.outputs.shell_versions }}
+      shell_versions: ${{ steps.filter-versions.outputs.shell_versions }}
+      plugins_version: ${{ steps.extract-plugins-version.outputs.plugins_version }}
 
     steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

```suggestion
          fetch-depth: 1
```
`fetch-depth: 0` will fetch the complete git history, which you probably do not need in this case?

---

probably not, will try with 1 for all occurences of checkout

---

yep, 1 is fine